packages/ckb-indexer/src/collector.ts

```diff
@@ -234,10 +241,33 @@ export class IndexerCollector implements BaseCellCollector {
     return counter;
   }
 
+  private async getLiveCellWithBlockHash(lastCursor?: string) {
+    let result: GetCellsResults = await this.getLiveCell(lastCursor);
+    const limit = pLimit(10);
+    //TODO refine this functions type
+    const inputs = result.objects.map((cell) => {
```

Comment:
found the JSON-RPC is [batchable](https://www.jsonrpc.org/specification#batch)，maybe we can implement via the batch spec.

Reply:
use json-rpc batch and test success both local and testnet rpc server